packages/account-v2/src/containers/ManualForm/ManualForm.tsx

```diff
@@ -31,8 +31,8 @@ export const ManualForm = ({
         const defaultValues = validationSchema.getDefault();
         const formValues = { ...defaultValues, ...formData };
         // Removing Selfie data from formValues as it is not part of this section of manual form
-        if (MANUAL_DOCUMENT_SELFIE in formValues) {
-            delete formValues[MANUAL_DOCUMENT_SELFIE];
+        if ('selfieWithID' in formValues) {
```

Comment:
MANUAL_DOCUMENT_TYPES.selfieWithID

Reply:
I cant use that as the form key is selfieWithID and not selfie_with_id